api/app/signals/apps/reporting/csv/datawarehouse/tasks.py

```diff
@@ -77,11 +77,12 @@ def zip_csv_files_endpoint(files: list):
 
 
 @app.task
-def save_and_zip_csv_files_endpoint():
+def save_and_zip_csv_files_endpoint(max_csv_amount: int = 30):
```

Comment:
Do we want the `max_csv_amount` to be a setting?

Reply:
yes it is overridden in django admin when you set a celery task, you can define parameters

Reply:
Ok